win32ss/user/ntuser/menu.c

```diff
@@ -380,7 +380,7 @@ MenuInit(VOID)
       ERR("MenuInit(): CreateFontIndirectW(hMenuFont) failed!\n");
       return FALSE;
     }
-    ncm.lfMenuFont.lfWeight = max(ncm.lfMenuFont.lfWeight + 300, 1000);
+    ncm.lfMenuFont.lfWeight = min(ncm.lfMenuFont.lfWeight + (FW_BOLD - FW_NORMAL), FW_HEAVY);
```

Comment:
Is there a reason why using FW_HEAVY instead of FW_BOLD?

Reply:
The system allows the user chooses bold weight fonts for normal menu font. This formula allows emboldening of bold fonts. `FW_HEAVY` is the maximum value.